### PR TITLE
Fix the vast-docker workflow

### PR DESCRIPTION
**Pull request by dominiklohmann**
###  :notebook_with_decorative_cover: Description

I have the feeling that the changes in #1208 are unrelated to the failure of this workflow, so let's run it whenever we touch our build system as well to catch issues earlier.

Edit: Confirmed to be broken; I'm repurposing this PR to fix the issue.

###  :memo: Checklist

- [x] All user-facing changes have changelog entries.
- [x] The changes are reflected on [docs.tenzir.com/vast](https://docs.tenzir.com/vast), if necessary.
- [x] The PR description contains instructions for the reviewer, if necessary.

### :dart: Review Instructions

n/t

### CODE REVIEW

**Comment by dominiklohmann** [comment]
[ch21472]

**Comment by 0snap** [comment]
This does not seem ready for review?

**Comment by dominiklohmann** [comment]
> This does not seem ready for review?

That is correct, should've been a draft. Sorry.

File: vast.yaml.example [hunk 1]
```diff
@@ -58,7 +58,7 @@ vast:
     self_sink:
       enable: true
       slice_size: 100
-      slice_type: 'arrow'
+      # slice_type: 'arrow'
```
Comment (mavam) [inline vast.yaml.example:61]:
```suggestion
      # slice_type: arrow
```

**Comment by mavam** [comment]
Are we building without Arrow support in Docker, and if so, why?

**Comment by dominiklohmann** [comment]
> Are we building without Arrow support in Docker, and if so, why?

Installing it on Buster is complicated, and a while ago it was impossible even. Nobody got around to changing it so far.

**Review by tobim — APPROVED** [review]
No need to test locally. Green CI should indicate that this can be merged.

**Pull request merged**